tests/org.polarsys.capella.vp.requirements.ju/src/org/polarsys/capella/vp/requirements/ju/preferences/KeepXHTMLTagsTest.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************

```

Comment:
Use the header as in file below (update year only):

https://github.com/eclipse/capella-requirements-vp/blob/491c5e93b4754a1690f98cf7c0741c07e6a14458/plugins/org.polarsys.kitalpha.vp.requirements.model.helpers/src/org/polarsys/kitalpha/vp/requirements/model/helpers/LabelHelper.java